Makefile.in

```diff
@@ -155,28 +155,28 @@ svn:
 	svn update
 
 autotools:
-	@sed -i.bak 's/.\/commit-version.sh/.\/version.sh/g' configure.ac
+	@sed -e 's/.\/commit-version.sh/.\/version.sh/g' configure.ac > configure.ac.tmp && mv configure.ac.tmp configure.ac
```

Comment:
I think it might be wise to add -f ("Do not prompt for confirmation before overwriting the destination path") to the mv commands, as I believe various things the user could do would end up with us prompting each time if we don't have the -f